manifests/application/migrator/daemon.pp

```diff
@@ -42,16 +42,16 @@
     $jsvc_version = 'latest',
     $owner = 'root',
     $group = 'root',
-    $service_name,
     $jsvc_exec = '$(which jsvc)',
     $java_home = undef,
+    $lib_dir = undef,
+    $arguments = {},
+    $jvmOptions = {},
+    $service_name,
```

Comment:
The changes to manifests/application/migrator/daemon.pp should be addressed in #104 which was just merged into master. Would you see if this can be reverted? If not, let me know and we can discuss.